.github/CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
 
 <h3>New features since last release</h3>
 
+* Adds the new template `AllSinglesDoubles` to prepare quantum states of molecules
+  using the `SingleExcitation` and `DoubleExcitation` operations. 
+  The new template reduces significantly the number of operations
+  and the depth of the quantum circuit with respect to the traditional UCCSD
+  unitary.
```

Comment:
Nothing to act on, but I must say, I am always still left with the amateur question: but does it do the same thing? I guess the answer is quite complicated!

Reply:
It does "essentially" the same thing :grin: 

The most compact way I can explain it is that when the UCCSD circuit is built from a qubit representation of fermionic operators, there are a lot of additional gates we must add that ensure the operators satisfy their anticommutation rules. But the effect this has on the state is just to add some phases. Here we forget about these phases, which during optimization can anyway be absorbed into the variational parameters. It makes the circuit much much simpler

Reply:
That means that the class of states it can prepare is essentially the same...but the cost landscapes will be vastly different, right? 

So the _behaviour_ of the template under optimisation, like which minima it finds and how fast, could be very different, no? This is why I much prefer the rename!

Reply:
I think that the class of states are similar. As @ixfoduap mentioned above, both templates are chemically inspired with UCCSD propagating the fermionic statistics to the circuits and AllSinglesDoubles sending this to be handled by the optimizer. Not sure is the cost landscapes will be vastly different.